Content.Server/Objectives/ObjectivesSystem.cs

```diff
@@ -114,25 +116,25 @@ private void AddSummary(ref string result, string agent, List<EntityUid> minds)
             if (title == null)
                 continue;
 
-            result += "\n";
-
             var custody = IsInCustody(mindId, mind) ? Loc.GetString("objectives-in-custody") : string.Empty;
 
             var objectives = mind.Objectives;
             if (objectives.Count == 0)
             {
-                result += Loc.GetString("objectives-no-objectives", ("custody", custody), ("title", title), ("agent", agent));
+                agentSummaries.Add((Loc.GetString("objectives-no-objectives", ("custody", custody), ("title", title), ("agent", agent)), 0f, 0));
                 continue;
             }
 
-            result += Loc.GetString("objectives-with-objectives", ("custody", custody), ("title", title), ("agent", agent));
+            var completedObjectives = 0;
+            var totalObjectives = 0;
+            var agentSummary = Loc.GetString("objectives-with-objectives", ("custody", custody), ("title", title), ("agent", agent));
 
             foreach (var objectiveGroup in objectives.GroupBy(o => Comp<ObjectiveComponent>(o).Issuer))
             {
                 //TO DO:
                 //check for the right group here. Getting the target issuer is easy: objectiveGroup.Key
                 //It should be compared to the type of the group's issuer.
-                result += "\n" + Loc.GetString($"objective-issuer-{objectiveGroup.Key}");
+                agentSummary += "\n" + Loc.GetString($"objective-issuer-{objectiveGroup.Key}");
 
```

Comment:
Use stringbuilder.

Reply:
Fixed, changed result parameter from ref string to StringBuilder as well